payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -551,6 +553,18 @@ class LpmRepository constructor(
             requirement = FpxRequirement,
             formSpec = LayoutSpec(sharedDataSpec.fields),
         )
+        PaymentMethod.Type.Alipay.code -> SupportedPaymentMethod(
+            code = "alipay",
+            requiresMandate = false,
+            mandateRequirement = MandateRequirement.Never,
```

Comment:
I think Till just removed this.

Reply:
Yep, it was removed 17 mins ago, just after I rebased.